### PR TITLE
Fix pagination alignment/451

### LINKED ISSUE #451: Tabular Data: Pagination alignment
ensure center vertical alignment on larger viewport sizes, and that all controls are the same height:

**Currently:**

screen shot 2018-09-18 at 3 56 40 pm

**Update**

screen shot 2018-09-18 at 3 51 47 pm

Center align and wrap controls on smaller viewports

**Currently**

screen shot 2018-09-18 at 3 59 07 pm

**Update**

screen shot 2018-09-18 at 3 53 42 pm

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #451 

Fixes the alignment for pagination and wraps pagination fields on mobile.  Also matches the height for input items and buttons.

### Screenshots

screen shot 2018-10-16 at 10 13 56 am
screen shot 2018-10-16 at 10 14 04 am

### Detailed test instructions:

1.  Visit `/wp-admin/admin.php?page=wc-admin#/analytics/products`
2.  Confirm that text and inputs are vertically aligned.
3.  On smaller screen sizes (<792px) confirm that pagination items wrap correctly.

### CODE REVIEW

File: client/components/pagination/style.scss [hunk 1]
```diff
@@ -48,28 +55,41 @@
...
 .woocommerce-pagination__page-picker {
-	margin-left: $spacing / 2;
-}
+	margin-left: $spacing;
 
-.woocommerce-pagination__page-picker-input {
-	margin-left: $spacing / 2;
-	width: 60px;
+	@include breakpoint( '<782px' ) {
+		margin-top: 14px;
```
Comment (Aljullu) [inline client/components/pagination/style.scss]: Looking at the InVision designs, I think that should be `16px` (you can use the variable `$gap`, instead).

File: client/components/pagination/style.scss [hunk 2]
```diff
@@ -48,28 +55,41 @@
...
+		box-shadow: none;
+	}
 }
 
 .woocommerce-pagination__per-page-picker {
-	margin-left: $spacing / 2;
+	margin-left: $spacing;
 
+	@include breakpoint( '<782px' ) {
+		margin-top: 14px;
```
Comment (Aljullu) [inline client/components/pagination/style.scss]: Same as before.

**Review by Aljullu — COMMENTED** [review]
That looks much better now, specially on mobile! :slightly_smiling_face: 

Small comment and LGTM.

File: client/components/pagination/style.scss (hunk 1 above)
Comment (timmyc) [reply client/components/pagination/style.scss]: FYI @joshuatf you should have gotten an invite to Invision in your gmail inbox 🌟

Comment (joshuatf) [reply client/components/pagination/style.scss]: @Aljullu Good catch on that and thanks for the `$gap` tip!
@timmyc Received access to InVision!

File: client/components/pagination/style.scss (hunk 2 above)
Comment (joshuatf) [reply client/components/pagination/style.scss]: Also fixed.

**Review by timmyc — APPROVED** [review]
:shipit:

**Pull request merged**